app/p/[slug]/features/project-details/components/sponsors/sponsors.tsx

```diff
@@ -0,0 +1,47 @@
+import { Avatar } from "components/ds/avatar/avatar";
+import { Link } from "components/ds/link/link";
+import { Flex } from "components/layout/flex/flex";
+import { Typography } from "components/layout/typography/typography";
+
+import { Section } from "../section/section";
+import { TSponsors } from "./sponsors.types";
+
+export function Sponsors({ sponsors }: TSponsors.Props) {
+  if (sponsors?.length === 0) {
+    return null;
+  }
+
+  return (
+    <Section
+      title={{
+        token: "v2.pages.project.overview.projectDetails.sponsors",
+        params: { count: sponsors?.length || 0 },
+      }}
+      remixIconName="ri-service-line"
+    >
+      <Flex wrap="wrap" className="gap-1">
+        {sponsors?.map(sponsor => (
```

Comment:
You already checked this before

```suggestion
        {sponsors.map(sponsor => (
```